examples/components/Portal.js

```diff
@@ -0,0 +1,87 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Ideally this `<Portal>` component should be replaced with something more fool-proof, like https://github.com/tajo/react-portal. This is in part why it got moved outside of the package.